control-plane/subcommand/api-gateway/command.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package apigateway
+
+import (
+	"context"
+	"flag"
+	"github.com/hashicorp/consul-k8s/control-plane/subcommand/common"
+	"github.com/hashicorp/consul-k8s/control-plane/subcommand/flags"
+	"github.com/mitchellh/cli"
+	"io"
+	"sync"
+)
+
+const (
+	defaultSDSServerHost = "consul-api-gateway-controller.default.svc.cluster.local"
```

Comment:
Just a note, if you compare this file to the original some of the fields are missing. This was to please the linter gods, I imagine we can add them back when we implement the command.